pegasus/sites.v3/hourofcode.com/public/learn/index.haml

```diff
@@ -1,10 +1,5 @@
 ---
 layout: wide_index
-social:
-  twitter:card: photo
-  twitter:site: "@codeorg"
-  og:image:width: '1200'
-  og:image:height: '630'
```

Comment:
This is getting set [here](https://github.com/code-dot-org/code-dot-org/blob/c25acaa3a0ce92bde1aef56ce04d8d05bede73d1/pegasus/src/social_metadata.rb#L234-L235) so I removed it.